praw/models/reddit/poll.py

```diff
@@ -0,0 +1,119 @@
+"""Provide poll-related classes."""
+from typing import Any, Optional
+
+from ..base import PRAWBase
+from ...util import cachedproperty
+
+
+class PollOption(PRAWBase):
+    """Class to represent one option of a poll.
+
+    If ``submission`` is a poll :class:`.Submission`, access the poll's
+    options like so:
+
+    .. code-block:: python
+
+       poll_data = submission.poll_data
+
+       # By index -- print the first option
+       print(poll_data.options[0])
+
+       # By ID -- print the option with ID '576797'
+       print(poll_data.get_option("576797"))
```

Comment:
Can we drop the `get_` prefix on this and call it just `option`?

Reply:
I think that this would be ambiguous, because an option method could create, delete, get, and/or update an option. I feel that perhaps implementing `__getitem__` would be a better way to do this.